osu.Game.Tests/Visual/Menus/TestSceneScreenNavigation.cs

```diff
@@ -112,6 +112,12 @@ public void TestSongContinuesAfterExitPlayer(bool withUserPause)
 
             AddStep("press enter", () => pressAndRelease(Key.Enter));
 
+            // Player loading may be blocked by either the cursor staying at 0,0 or a focused overlay being opened taking away hover from player loader
```

Comment:
the 0,0 cursor thing sounds like a bug or regression?

Reply:
need to rename this comment, the `0,0` bug has been already fixed by the framework-side PR but I was referring to the toolbar as well.